Dockerfile.rhel7

```diff
@@ -1,11 +1,11 @@
-FROM registry.svc.ci.openshift.org/ocp/builder:rhel-8-golang-1.15-openshift-4.6 AS builder
+FROM registry.ci.openshift.org/ocp/builder:rhel-8-golang-1.16-openshift-4.9 AS builder
 RUN yum install -y libvirt-devel
 
 WORKDIR /go/src/github.com/openshift/cluster-api-provider-libvirt
 COPY . .
 RUN go build -o machine-controller-manager ./cmd/manager
 
-FROM registry.svc.ci.openshift.org/ocp/4.6:base
+FROM registry.ci.openshift.org/ocp/4.9:base
 RUN INSTALL_PKGS=" \
```

Comment:
These changes are the same as https://github.com/openshift/cluster-api-provider-libvirt/pull/222/commits/4156429e65cfa60b1491c6c2ec043752c4bb31f9 
(not an objection, just linking the 2 PRs)